test/e2e/flowaggregator_test.go

```diff
@@ -259,18 +266,49 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 						t.Errorf("Record does not have correct egressNetworkPolicyNamespace")
 					}
 				}
-				// Check the bandwidth using octetDeltaCount in data record.
+				curOctetTotalCount := getUnit64FieldFromRecord(t, record, "octetTotalCountFromSourceNode")
+				if curOctetTotalCount > octetTotalCount {
+					octetTotalCount = curOctetTotalCount
+				}
```

Comment:
Grouping these lines with bandwidth check code that uses octectTotal field.

Reply:
Thanks, I grouped these lines with bandwidth check code that uses octectDelta field. Becasue average bandwidth check code is using the max value of octectTotalCount.